src/Reader.php

```diff
@@ -210,6 +210,8 @@ public function setDefaultValueBinder(): self
      */
     public function loadSpreadsheet($import)
     {
+        $this->beforeImport($import);
```

Comment:
I think we actually cannot do this, as this would break people depending on the spreadsheet already being loaded in memory.

I'm fine with adding a before reading event instead

Reply:
I'm agree with `beforeReading` event

Reply:
@patrickbrouwers do you have a chance to add new event?

Reply:
You can do this in this PR

Reply:
Ok, I'll add a new event in this PR